src/Map.ts

```diff
@@ -465,7 +480,17 @@ export class Map extends maplibregl.Map {
       }
     });
 
-    const terrainCallback = (evt: any) => {
+    this.once("style.load", () => {
+      if (typeof options.minimap === "object") {
+        this.minimap = new Minimap(options.minimap, options);
```

Comment:
Passing the `options` object that was originally intended for the main map will result in the minimap's Map showing all sort of controls that are not relevant (scale, fullscreen, terrain) and also some options that may result in an unexpected behavior: if `hash` is true, then the URL's hash is written by two maps and it's not always the main map that prevails so when I reload, I see my main map with the settings of the last minimap.

I think only some options of the main Map's constructor should be passed.

Reply:
When I build `this.#options` inside the constructor of `Minimap` it overrides where necessary. I liked `hash` so I kept it but can you go over that section and come up with a list of items you don't want to see be inherited?